gst/droidcamsrc/gstdroidcamsrc.c

```diff
@@ -120,6 +122,10 @@ static guint droidcamsrc_signals[LAST_SIGNAL];
 #define DEFAULT_IMAGE_MODE             GST_DROIDCAMSRC_IMAGE_MODE_NORMAL
 #define DEFAULT_TARGET_BITRATE         12000000
 #define DEFAULT_POST_PREVIEW           FALSE
+/* JPEG quality for photos in %, min 10% max 100% - safe values */
+#define DEFAULT_JPEG_QUALITY           100
```

Comment:
To me 100% jpeg quality by default sounds much, is there really need?

Reply:
Looking at the Android source, 85-90 seems to be a 'superfine' setting that it might be more sensible to default to.

Reply:
Seems that 90% was default in AOSP since 2012. Now in 2021 the most of camera app in Android set it to default 100%, 'cause it could increase jpeg quality even on devices with cheap cameras.

Reply:
On my generic experience with jpeg quality parameters the >90% values have been a good way to increase file size without much visual effect. And bad camera will be a bad camera even with lossless image files.

Reply:
I agree that it is hard to feel grahic difference between them. But when people compare android photo and sailfish photo on same device they can see difference in size and think that sailfish photos 2 times worse quality than on Android.

Reply:
So my suggestion for now is to keep it at 100% and add ability to change quality dynamically in jolla-gallery or configuration file.

Reply:
> they can see difference in size and think that sailfish photos 2 times worse quality than on Android.

Eh, so we would be increasing the file size just to give impression on good quality?

Reply:
More or less, actual quality will increase also, but could not be compared with increased size :)